internal/db/migrations/2025-01-22.1-remove_tss_tables.sql

```diff
@@ -0,0 +1,37 @@
+-- +migrate Up
+
+DROP INDEX IF EXISTS idx_tx_current_status;
+DROP INDEX IF EXISTS idx_claimed_until;
+DROP TABLE IF EXISTS tss_transactions;
+
+DROP INDEX IF EXISTS idx_original_transaction_hash;
+DROP INDEX IF EXISTS idx_last_updated;
+DROP TABLE IF EXISTS tss_transaction_submission_tries;
+
```

Comment:
FYI: if you were to drop the table with the `CASCADE` option, all dependent objects – including indexes, constraints, and triggers – are automatically dropped as well.